src/api/queries/arbeidsgiver/oppfolgingsplanerQueriesAG.ts

```diff
@@ -67,3 +69,49 @@ export const useTidligereOppfolgingsplanerAG = () => {
     tidligereOppfolgingsplaner: [],
   };
 };
+
+export const useOpprettOppfolgingsplanAG = () => {
+  const sykmeldt = useDineSykmeldte();
+  const oppfolgingsplaner = useOppfolgingsplanerAG();
+
+  const apiBasePath = useApiBasePath();
+  const apiOppfolgingsplanPath = useOppfolgingsplanApiPath();
+  const queryClient = useQueryClient();
+
+  const opprettOppfolgingsplan = async (data: {
+    kopierTidligerePlan: boolean;
+  }) => {
+    if (!oppfolgingsplaner.isSuccess || !sykmeldt.isSuccess) {
+      return;
+    }
+
+    const opprettOppfoelgingsdialog: OpprettOppfoelgingsdialog = {
```

Comment:
Skal vi forsøke å gå vekk fra "dialog"-begrepet?

Reply:
Ja, absolutt.

Reply:
Jeg ser nå at den `OpprettOppfoelgingsdialog` er skjema for backend-kallet, og i backend heter det `RSOpprettOppfoelgingsdialog` og da tenker jeg at den kanskje bør hete det, men jeg kan rename variabel-navnet til `opprettOppfoelgingsplan`